spec/unit/server_create_command_spec.rb

```diff
@@ -110,89 +108,13 @@ class ServerCreate < Chef::Knife::Cloud::ServerCreateCommand
     include Chef::Knife::Cloud::ServerCreateOptions
   end
 
-  describe "#bootstrap options" do
-
-    it "set chef config knife options" do
+  describe "Bootstrap Protocol option" do
+    it "not to be set in chef config knife options" do
       instance = ServerCreate.new
-      bootstrap_url = "bootstrap_url"
-      bootstrap_install_command = "bootstrap_install_command"
-      bootstrap_wget_options = "bootstrap_wget_options"
-      bootstrap_curl_options = "bootstrap_curl_options"
-      bootstrap_no_proxy = "bootstrap_no_proxy"
-
-      instance.options[:bootstrap_url][:proc].call bootstrap_url
-      expect(Chef::Config[:knife][:bootstrap_url]).to eq(bootstrap_url)
-
-      instance.options[:bootstrap_install_command][:proc].call bootstrap_install_command
-      expect(Chef::Config[:knife][:bootstrap_install_command]).to eq(bootstrap_install_command)
-
-      instance.options[:bootstrap_wget_options][:proc].call bootstrap_wget_options
-      expect(Chef::Config[:knife][:bootstrap_wget_options]).to eq(bootstrap_wget_options)
-
-      instance.options[:bootstrap_curl_options][:proc].call bootstrap_curl_options
-      expect(Chef::Config[:knife][:bootstrap_curl_options]).to eq(bootstrap_curl_options)
-
-      instance.options[:bootstrap_no_proxy][:proc].call bootstrap_no_proxy
-      expect(Chef::Config[:knife][:bootstrap_no_proxy]).to eq(bootstrap_no_proxy)
-
-      expect(instance.options[:auth_timeout][:default]).to eq(25)
-    end
-  end
-
-  describe "#before_bootstrap" do
-
-    before(:all) { @instance = ServerCreate.new }
-
-    context "bootstrap_protocol shh" do
-      before { @instance.config[:bootstrap_protocol] = "ssh" }
+      bootstrap_protocol = "bootstrap_protocol"
```

Comment:
I'd probably change this to something like `bootstrap_protocol = "nothing"` or such. it's slightly confusing having the value the same as the variable name.